CLASSES/class_HTML.php

```diff
@@ -175,7 +186,18 @@ function __construct()
                     }
                     $this->result['feedName'] = ConfigBroker::getConfig('Site Name', 'CCHits.net') . ' - ' . ConfigBroker::getConfig('Daily Show Name', 'Daily Exposure Show');
                 }
-                $this->daily($object[1]);
```

Comment:
$object[1] is initialized with $this->arrUri['path_items'][1]. But since this array may have been modified, I believe what we really want as argument of $this->daily() is the actual value of arrUri['path_items'][1].
I agree, it should probably be done for weekly and monthly too.